specification/maps/data-plane/Timezone/preview/1.0/timezone.json

```diff
@@ -209,6 +212,7 @@
           },
           {
             "name": "query",
+            "x-ms-client-name": "coordinate",
             "in": "query",
```

Comment:
This should be collectionFormat csv with minItems and maxItems both set to 2 and type set to number with format float

Reply:
Updated